### ISSUE TITLE
JAX 0.4.36 Failing

### ISSUE THREAD

**Issue by lockwo** [patrick-kidger/diffrax#532]
With the latest release of jax (1 hour ago) the latest version of diffrax fails. https://github.com/patrick-kidger/diffrax/pull/527 looks like it fixes the problem (at least the one I encountered), but it would be great to get an official release (or disallow this version of jax).

**Comment by lockwo** [comment]
I stand corrected, https://github.com/patrick-kidger/diffrax/blob/dev/test/test_adjoint.py#L231 seems to also be failing 

```
test_adjoint.py:231:44 - error: "CustomVJPException" is not a known member of module "jax.interpreters.ad" 
```

**Comment by patrick-kidger** [comment]
Closing as resolved with the 0.6.1 release of Diffrax!
